api/WEBGL_compressed_texture_s3tc.json

```diff
@@ -6,22 +6,17 @@
         "support": {
           "chrome": [
             {
-              "version_added": true
-            },
-            {
-              "version_added": true,
-              "prefix": "WEBKIT_"
-            }
-          ],
-          "chrome_android": [
-            {
-              "version_added": true
+              "version_added": "26"
             },
             {
               "version_added": true,
               "prefix": "WEBKIT_"
             }
           ],
+          "chrome_android": {
+            "version_added": true,
```

Comment:
Does this mean it was unprefixed in Chrome desktop but remains prefixed-only on Android? That sounds quite unusual, can you double check?

Reply:
I should have checked the prefixed support in Android as well; it looks like this API isn't supported in Chrome Android at all.  Fixing the data.